### PR TITLE
Exercise

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: lesson4/public/tickets.html [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-  <div>
-    <pre id= 'displaysya'></pre>
-  </div>
-  <script>
-    //send the # of tickets bought
-    const button = document.getElementById('theButton');
-    button.onclick = () =>{
-    const quantity = document.getElementById('#Tickets').value;
-    if(quantity<=0){
+    <p>Saumya's Tickets</p>
```
Comment (allopez7) [inline lesson4/public/tickets.html]: Saumya's tickets?

**Review by allopez7 — APPROVED** [review]
Looks good!

File: lesson4/public/indexmove.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  <div id = 'pic'>
+    <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSnfpzoD-_HTIGH37ncMqaYiqjOI4MrXSPSTCyAHbJdSsc6O9vP">
+  </div>
+  </div>
+  <script>
+
+//Going up
+    const goUp = document.getElementById('up');
+    goUp.onclick = ()=>{
+      const t = 'up'
```
Comment (songz) [inline lesson4/public/indexmove.html:22]: please use better variable names throughout your file. Also, using a helper function here would help alot so you don't repeat your code.

File: lesson4/move.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+app.listen(3618);
+app.use(express.static('public'));
+
+
+let vertical = 0;
+let horizontal = 0;
+
+app.get('/moving', (req, res) =>{
+  const data = req.query.a;
+  if(data == 'up'){
```
Comment (songz) [inline lesson4/move.js:12]: look up switch method. see you can integrate it!

**Pull request merged**